test/library/standard/Socket/connectblocking.chpl

```diff
@@ -0,0 +1,148 @@
+use UnitTest;
+use Socket;
+
+proc test_connection_ipv4_dns(test: borrowed Test) throws {
+  var port:uint(16) = 8811;
+  var host = "127.0.0.1";
```

Comment:
Not actually using a DNS name here - do you want to use `localhost` here for this test?